app/views/provider/admin/accounts/new.html.slim

```diff
@@ -1,6 +1,11 @@
 h2 Create new Account
 = semantic_form_for @provider, builder: Fields::FormBuilder, url: provider_admin_accounts_path do |form|
 
+  - if @provider.errors.any?
+    ul style = "color: red"
```

Comment:
We shouldn't be adding custom error messages, but rather use the default form validation styles.

![validation](https://user-images.githubusercontent.com/1270328/190641783-8313563c-be67-4586-b9bb-4decb878b3ee.png)

Apparently, the validation here is on the internal field (`domain` and `self_domain` I guess), that's why the highlighting doesn't work.

Reply:
@mayorova  This issue is for tenant (account in master portal) and not for account in admin portal. The screenshot you attached is from admin portal?

Reply:
@nidhi-soni1104 The screenshot is from the master portal, the form creating a tenant.